src/components/MessagesList/MessagesGroup/Message/MessagePart/FilePreview.vue

```diff
@@ -472,44 +478,33 @@ export default {
 			event.stopPropagation()
 			event.preventDefault()
 
-			let permissions = ''
-			if (this.permissions) {
-				if (this.permissions & OC.PERMISSION_CREATE) {
-					permissions += 'CK'
-				}
-				if (this.permissions & OC.PERMISSION_READ) {
-					permissions += 'G'
-				}
-				if (this.permissions & OC.PERMISSION_UPDATE) {
-					permissions += 'W'
-				}
-				if (this.permissions & OC.PERMISSION_DELETE) {
-					permissions += 'D'
-				}
-				if (this.permissions & OC.PERMISSION_SHARE) {
-					permissions += 'R'
+			const fileInfo = this.generateViewerObject(this)
+
+			if (this.isSharedItemsTab && this.sharedItemsType === SHARED_ITEM.TYPES.MEDIA) {
```

Comment:
Maybe it's also good to have it for the chat messages as well?

Reply:
BTW, `isSharedItemsTab `should be renamed because it is also true in `SharedItemsBrowser` . Maybe isOverviewGrid or sthg similar

Reply:
> Maybe it's also good to have it for the chat messages as well?

I personally don't think, that it should be implemented across all messages, only within a single one. But that's for NC28 also, as we don't have multiple items in one message.

Reply:
> `isSharedItemsTab `should be renamed

It's a part of different PR, i could fix it there, to not raise conflicts

Reply:
> > Maybe it's also good to have it for the chat messages as well?
> 
> I personally don't think, that it should be implemented across all messages, only within a single one. But that's for NC28 also, as we don't have multiple items in one message.

I actually think that it should be implemented across all messages ( for easier media navigation when checking the chat). It is common in other apps/websites.

Reply:
> it should be implemented across all messages 

I'd discuss it in public or design channel maybe